client-v2/src/app/models/defaults.ts

```diff
@@ -0,0 +1,8 @@
+import { EntityType } from './entities.model'
+
+export const ENTITY_NAME_DEFAULTS: Record<EntityType, string> = {
+    [EntityType.TASKLIST]: 'Untitled tasklist',
+}
+
+// @TODO: remove placeholder
+export const DEFAULT_TASKLIST_NAME = ENTITY_NAME_DEFAULTS[EntityType.TASKLIST]
```

Comment:
This can be removed and all existing references can be updated to use the more explicit `ENTITY_NAME_DEFAULTS[EntityType.TASKLIST]`